docs/Process_Merge-Policy.md

```diff
@@ -15,80 +16,86 @@ Kernel and U-Boot maintainers are usually grouped by SoC Architecture.
 
 Supported boards will have the most scrunity with code review.
 
-## U-Boot Patches ##
+## U-Boot Patches
+
+- Standard contributors provide _tested-by_ submitter (`armbianmonitor -u` with a fresh build).
+- SoC maintainer maybe submit a PR with only a reviewed by of the lead SoC maintainer.
 
-- Standard contributors provide _tested-by_ submitter (`armbianmonitor -u` with a fresh build)
-- SoC maintainer maybe submit a PR with only a reviewed by of the lead SoC maintainer
+## Kernel Related Patches
 
-## Kernel Related Patches ##
+### legacy and current branches
 
-### legacy and current branches ###
+- DT changes reviewed by at least one person familiar with this SoC (lead maintainer or deputy), _tested-by_ the contributor who sends it (armbianmonitor).
+- Trivial module activation doesn't matter.
 
-- DT changes reviewed by at least one person familiar with this SoC (lead maintainer or deputy), _tested-by_ the contributor who sends it (armbianmonitor)..
-- trivial module activation doesn't matter
+### dev branches
 
-### dev branches ###
+Constraints are at the discretion of the SoC mantainer.  This builds are not expected to be stable.
 
-Constraints are at the discretion of the SoC mantainer. This builds are not expected to be stable.
+## Armbian Build Scripts
 
-## Armbian Build Scripts ##
+This pertains to code used to build system images, OS configuration, and supporting packages (basically anything not u-boot or kernel source).
```

Comment:
Actually it is called `U-Boot` and not u-boot: https://www.denx.de/wiki/U-Boot

Reply:
I changed a few (because it is both ways different places in docs it seems) but then I started to wonder about that and that's actually why I stopped.

If you want to just skip it, I can do another PR just about that if you like.  It will be easy to do something like projectile replace in project on all occurances.  But I would manually double check each one and leave any ones in examples as lower `u-boot` for instance.